### ISSUE TITLE
VirtualizedSelect is undefined

### ISSUE THREAD

**Issue by armed** [bvaughn/react-virtualized-select#55]
Hi, thanks for Virtualized and VirtualizedSelect. Great libs!

I wasn't able to use VS as UMD, by including it in browser:
```
// react and friends is already loaded in my app bundle.
<script src="https://unpkg.com/react-virtualized-select@2.4.0/dist/umd/react-virtualized-select.js"></script>
```

What I'm missing?

**Comment by bvaughn** [comment]
Looks like the UMD build is broken. I guess it's so infrequently used that no one had noticed until now.

You can patch-fix it temporarily like so:
```js
const VirtualizedSelect = window['react-virtualized-select'].default;
```

I'll try to put out a new release at some point when I find time. If you beat me to it though I'll happily review a PR.

**Comment by bvaughn** [comment]
Thanks for the PR!

**Comment by bvaughn** [comment]
Published as [2.4.1](https://github.com/bvaughn/react-virtualized-select/blob/master/CHANGELOG.md#241)

**Comment by armed** [comment]
Hey, Brian

Thanks!
